_data/rules.yml

```diff
@@ -292,3 +292,11 @@ services:
       - name: prometheus/blackbox_exporter
         doc_url: https://github.com/prometheus/blackbox_exporter
         rules:
+          - name: Status Code
+            description: HTTP status code isn't equal to 200
+            query: 'probe_http_status_code != 200'
+            severity: error
+          - name: SSL Cert expiration
```

Comment:
Could you please duplicate this alert with `error` severity for expired certificates ?

Reply:
done